app/v1/connectors/BaseDesConnector.scala

```diff
@@ -28,12 +28,16 @@ trait BaseDesConnector {
   val http: HttpClient
   val appConfig: AppConfig
 
-  val logger = Logger(this.getClass)
+  val logger: Logger = Logger(this.getClass)
 
   private[connectors] def desHeaderCarrier(implicit hc: HeaderCarrier): HeaderCarrier =
     hc.copy(authorization = Some(Authorization(s"Bearer ${appConfig.desToken}")))
       .withExtraHeaders("Environment" -> appConfig.desEnv)
```

Comment:
This API only has one endpoint so this method (as well as the HTTP methods for GET, PUT, and DELETE in this file) will be unused. Consider just adding the OriginatorId to this method instead of adding a second one.

Reply:
Yh, I created a new one cause I thought it was used for the sample endpoint but it shouldn't matter as the sample will get removed anyway. Will update it to that and remove the others which are unused anyway. Thanks